axonserver/src/main/java/io/axoniq/axonserver/grpc/SubscriptionQueryRequestTarget.java

```diff
@@ -106,13 +106,15 @@ public void onError(Throwable t) {
         if (!subscriptionQuery.isEmpty()) {
             unsubscribe(subscriptionQuery.get(0));
         }
+        responseObserver.onError(t);
```

Comment:
Darn, this was it? Such a small thing to forget....

Reply:
Yes, don't tell me :)